src/openapi.yaml

```diff
@@ -17,6 +17,7 @@ paths:
         required: false
         schema:
           type: string
+          format: uri
```

Comment:
There are no `maxLength` and `minLength` properties for uri query params in openzaak schemas. 

And we don't have checks to make sure, that the query param fits these requirements. Should I change implementation and add this check?

Reply:
I think steven added the same thing for Open Zaak or VNG APIs. So, yes. It will provide clearity on how long a URI can be and that is indeed required.